### PR TITLE
travis.yml: make yamllint compliant

### PR DESCRIPTION
**Pull request by waltdisgrace**
Addresses stratis-storage/project#173

### CODE REVIEW

File: src/dbus_client_gen/_managed_objects_queries.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def mo_query_builder(spec):
         property_names = frozenset(p.attrib["name"] for p in spec.findall("./property"))
     except KeyError as err:  # pragma: no cover
         fmt_str = (
-            "No name attribute found for some property belonging to " 'interface "%s"'
+            "No name attribute found for some property belonging to 'interface \"%s\"'"
```
Comment (mulkieran) [inline src/dbus_client_gen/_managed_objects_queries.py:126]: Better to drop the single quotes as I did in https://github.com/stratis-storage/dbus-python-client-gen/pull/45.

**Comment by bgurney-rh** [comment]
The travis-ci build passed; Travis never reported the status to GitHub.

**Comment by waltdisgrace** [comment]
@mulkieran The build passed: https://github.com/stratis-storage/dbus-signature-pyparsing/pull/14

File: src/dbus_client_gen/_managed_objects_queries.py (hunk 1 above)
Comment (mulkieran) [inline src/dbus_client_gen/_managed_objects_queries.py:126]: Please remove single quotes. Please do not force push again, but add your fixup commits to the existing PR.

**Comment by mulkieran** [comment]
Looks ready, except for yamllint target, in Pending.

**Pull request closed without merging**